test/OpenTelemetry.Instrumentation.AspNetCore.Tests/RouteTests/RoutingTestCases.json

```diff
@@ -198,5 +198,15 @@
     "expectedStatusCode": 200,
     "currentHttpRoute": null,
     "expectedHttpRoute": "/MinimalApiUsingMapGroup/{id}"
+  },
+  {
+    "name": "Exception Handled by Exception Handler Middleware",
+    "minimumDotnetVersion": 7,
```

Comment:
I think you can remove this line. This should also work for .NET 6, right?

Reply:
done